styles/Home.module.css

```diff
@@ -114,3 +114,11 @@
     flex-direction: column;
   }
 }
+
+.SignIn {
+  text-align: center;
+}
+
+.BoldText {
+  font-weight: bolder;
+}
```

Comment:
and empty line at the end please

can we use classes with camel casing

Reply:
yep my bad i realized too late  to add an extra line and make it camel case